prow/git/v2/executor.go

```diff
@@ -64,12 +65,14 @@ type censoringExecutor struct {
 
 func (e *censoringExecutor) Run(args ...string) ([]byte, error) {
 	logger := e.logger.WithField("args", strings.Join(args, " "))
+	now := time.Now()
 	b, err := e.execute(e.dir, e.git, args...)
 	b = e.censor(b)
 	if err != nil {
 		logger.WithError(err).WithField("output", string(b)).Debug("Running command failed.")
 	} else {
 		logger.Debug("Running command succeeded.")
 	}
+	logger.WithFields(logrus.Fields{"duration": time.Since(now), "dir": e.dir}).Info("Time taken to execute command.")
```

Comment:
How about adding another field for the git subcommand that is used?

Reply:
That is in args